bos

```diff
@@ -1478,6 +1479,7 @@ prog
           is_omitting_message_from: options.messageOmitFromKey,
           lnd: (await lndForNode(logger, options.node)).lnd,
           max_fee: options.maxFee,
+          max_fee_rate: options.maxFeeRate || undefined,
```

Comment:
if the user passes zero here, it should use that number

otherwise the user might think they are putting zero max fee rate and unexpectedly pay a fee